python/main.py

```diff
@@ -98,8 +111,66 @@ async def get_image(image_name):
 class Item(BaseModel):
     name: str
     category: str
+    image_name: str
 
 
-def insert_item(item: Item):
+def insert_item(item: Item, image_name: str):
     # STEP 4-1: add an implementation to store an item
-    pass
+    try:
+        if not JSON_DB.exists():
+            with open(JSON_DB, "w", encoding="utf-8") as f:
+                json.dump({"items": []}, f, indent=2)
```

Comment:
Can we extract the default value of the JSON file to a separate variable. It might be useful